site2/website-next/src/pages/contributing.md

```diff
@@ -201,6 +201,10 @@ You'll then push to your branch on GitHub. Note: when updating your commit after
 
 Navigate to the [Pulsar GitHub Repo](https://github.com/apache/pulsar) to create a pull request.
 
+> **Note**
+> 
+> Vague, boring, and unclear PR titles decrease team efficiency and productivity. Good titles speed up the review process and increase search efficiency. PR titles should be engaging, easy to understand, and readable. For how to **write self​-explanatory PR titles** and **quality PR title examples**, see [[Guideline] Pulsar PR Naming Convention](https://docs.google.com/document/d/1d8Pw6ZbWk-_pCKdOmdvx9rnhPiyuxwq60_TrD68d7BA/edit#).
+
```

Comment:
Was it intentional to make this change as part of the PR with the title "[improve][website] Update Pulsar WeChat account ID" ? 🤔

Reply:
Sorry I don't understand what you mean. Do you mean I need to rename this PR title because I submit two different changes?